proto/sign/signingMessages.go

```diff
@@ -109,58 +110,77 @@ func (sm *SigningMessage) UnmarshalBinary(data []byte) error {
 	return protobuf.DecodeWithConstructors(data, sm, cons)
 }
 
+type JSONdata struct{
+		Len int64
+		Data []byte
+}
+
```

Comment:
We should find the approriate place to put that struct instead of declaring it two times.
What about coconet/signingMessages ? for the moment at least.